packages/now-cli/src/util/projects/find-projects-for-domain.ts

```diff
@@ -0,0 +1,46 @@
+import chalk from 'chalk';
+import Client from '../client';
+import wait from '../output/wait';
+import { Project } from '../../types';
+import { URLSearchParams } from 'url';
+
+export async function findProjectsForDomain(
+  client: Client,
+  domainName: string
+): Promise<Project[] | Error> {
+  const cancelWait = wait(
+    `Searching project for domain ${chalk.bold(domainName)}`
+  );
+  try {
+    const limit = 50;
+    let result: Project[] = [];
+
+    const query = new URLSearchParams({
+      hasProductionDomains: '1',
+      limit: limit.toString(),
+      domain: domainName,
+    });
+
+    for (;;) {
```

Comment:
```suggestion
    for (let i=0; i<1000; i++) {
```

Should we add an upper limit just to be safe and avoid infinite loops?